arch/unitroot/unitroot.py

```diff
@@ -1714,3 +1714,63 @@ def kpss_crit(stat, trend='c'):
     crit_value = interp(cv, y[::-1], x[::-1])
 
     return pvalue, crit_value
+
+  def BandWidthNW(y, kernel = "ba"):
+    '''
+    | Automatic bandwidth selection of Andrews (1991) and of Newey and West (1994)
+    | 
+    | y: Data on which to apply the bandwidth selction
+    | kernel: The kernel function to use for selecting the bandwidth
+    |  - "ba": Bartlett kernel (default value)
+    |  - "pa": Parzen kernel
+    |  - "qs": Quadratic Spectral kernel
+    | 
+    | return the Bandwidth
+    '''
+
+    n = int(4 * np.power(len(y)/100, 2/9))       
+    sig = (n+1)*[0]
+
+    for i in range(n+1):
+
```

Comment:
I prefer tigheer spacing.  Only use empty lines sparingly to separate logical blocks of code, not with, e.g., a loop